var/phpstan-utilities/console-application.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+require __DIR__.'../../../config/bootstrap.php';
+$kernel = new \App\Kernel('phpstan', (bool) $_SERVER['APP_DEBUG']);
```

Comment:
No need to ne able to detect the debug mode from $_SERVER. You can hardcode `true`
```suggestion
$kernel = new \App\Kernel('phpstan', true);
```